_shared/project/tox.ini

```diff
@@ -25,13 +25,19 @@ skip_install =
 setenv =
     PYTHONUNBUFFERED = 1
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
+    SQLALCHEMY_SILENCE_UBER_WARNING=1
 {% if cookiecutter.get("postgres") == "yes" %}
     dev: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/postgres}
-    tests,functests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_test}
+    tests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_tests}
+    functests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_functests}
+    dev: SENTRY_ENVIRONMENT = {env:SENTRY_ENVIRONMENT:dev}
```

Comment:
Set the `SENTRY_ENVIRONMENT` envvar to `dev` in dev. This is useful if testing with Sentry locally and LMS had it.